docs/book/src/forc/workspaces.md

```diff
@@ -25,6 +26,23 @@ members = ["member1", "path/to/member2"]
 The `members` field accepts entries to be given in relative path with respect to the workspace root.
 Packages that are located within a workspace directory but are *not* contained within the `members` set are ignored.
 
+## The `[patch]` section
+
+The `[patch]` section can be used to override any dependency in the workspace dependency graph. The usage is the same with package level `[patch]` section and details can be seen [here](./manifest_reference.md#the-patch-section).
```

Comment:
Even though it may seem obvious to us, I think we might want to add a note here that this will only take effect for packages that are declared as members. The [AMM project](https://github.com/FuelLabs/sway-applications/blob/master/AMM/project/Forc.toml) for example does not declare `libraries` as a member.